src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -35,17 +37,23 @@ class ProjectCard extends React.Component {
   render() {
     const {project, stats, params} = this.props;
 
+    const bookmarkText = project.isBookmarked
+      ? t('Unstar this project')
```

Comment:
We use "Add to/Remove from bookmarks" on the projects/teams page

Reply:
👍 will update to match